core/src/nodes/node.rs

```diff
@@ -347,32 +346,37 @@ where
     }
 }
 
-// TODO:
-/*impl<TTrans> fmt::Debug for NodeEvent<TTrans>
-where TTrans: Transport,
-      <TTrans::Listener as Stream>::Error: fmt::Debug,
+impl<TMuxer, TUserData> fmt::Debug for NodeEvent<TMuxer, TUserData>
+where 
```

Comment:
```
> git diff --check master 
core/src/nodes/node.rs:350: trailing whitespace.
+where 
```

Reply:
Our [.editorconfig](https://github.com/libp2p/rust-libp2p/blob/master/.editorconfig) specifies `trim_trailing_whitespace=true`. I wonder why it is not picked up?